packages/text-area/src/TextArea.tsx

```diff
@@ -122,10 +123,10 @@ const colorSets: Record<Mode, ColorSets> = {
   },
 };
 
-type BaseTextAreaProps = HTMLElementProps<'textarea', never> & {
+type BaseTextAreaProps = HTMLElementProps<'textarea', HTMLTextAreaElement> & {
   id?: string;
   darkMode?: boolean;
-  label: string;
+  label: string | null;
```

Comment:
Can we be consistent with TextInput and make this optional as well?

Reply:
yep